helm/values.yaml

```diff
@@ -126,5 +95,4 @@ logConfig:
 config-bootstrapper:
   job:
     prefix: entity
-
-configurationCommands: {}
+  configurationCommands: {}
```

Comment:
is this an indentation issue?

Reply:
@kotharironak  , no its intentional. `configurationCommands` value now needs to be passed to config-boostrapper helm chart.